app/src/test/java/com/kickstarter/ui/activities/compose/OnboardingFlowScreenTest.kt

```diff
@@ -117,69 +117,72 @@ class OnboardingFlowScreenTest : KSRobolectricTestCase() {
     }
 
     @Test
-    fun `Test save projects page next button click skips notifications page if device doesn't need notification permissions`() {
-        setupOnboardingScreen(deviceNeedsNotificationPermissions = false)
+    fun `Test activity tracking page primary button click invokes allowTracking`() {
+        setupOnboardingScreen()
 
         primaryButton.performClick() // Welcome -> Save
         Thread.sleep(500)
         primaryButton.performClick() // Save -> Activity Tracking
+        Thread.sleep(500)
```

Comment:
nitpick no need to change anything: not fan of Thread.sleep(), usually `advanceUntilIdle()` there are a few samples of it in `LocationSheetTest` where there is a debounce operator in place -> https://github.com/kickstarter/android-oss/blob/8abb8dbd85e7ed5fcb523e48cb68701e60f2557b/app/src/test/java/com/kickstarter/ui/activities/compose/search/LocationSheetTest.kt#L180